common/transport/amqp/typings.json

```diff
@@ -1,8 +1,4 @@
 {
   "name": "azure-iot-amqp-base",
-  "main": "index.d.ts",
-  "dependencies": {
-    "amqp10": "registry:npm/amqp10#3.3.1+20170124180914",
-    "bluebird": "registry:npm/bluebird#3.5.0+20170314181206"
-  }
+  "main": "index.d.ts"
 }
```

Comment:
I think we can remove this file alltogether.

Reply:
Done.

---
In reply to: [209123932](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209123932) [](ancestors = 209123932)